src/wasm/wasm-validator.cpp

```diff
@@ -3861,7 +3861,7 @@ static void validateGlobals(Module& module, ValidationInfo& info) {
     auto globalFeats = g->type.getFeatures();
     if (!info.shouldBeTrue(globalFeats <= module.features, g->name, "")) {
       info.getStream(nullptr)
-        << "global type requires additional features "
+        << "global type requires additional features"
```

Comment:
don't we want a space, so it is `"features [--foo]"` and not `"features[--foo]"`?

Reply:
Ah, this applies to other uses of `getMissingFeaturesList` but not this one. Fixing.